ferrocene/doc/qualification-plan/src/standards/iso-26262.rst

```diff
@@ -8,157 +8,209 @@ ISO 26262 requirements
    :header-rows: 1
 
    * - Clause
-     - Document chapter(s)
+     - ``rustc``
+     - ``rustfmt``
 
    * - 11.4.1
      - See the traceability below
+     - See the traceability below
 
    * - 11.4.2
      - N/A
+     - N/A
 
    * - 11.4.3
-     - :doc:`Safety Manual - Usage <safety-manual:rustc/usage>`
+     - :doc:`Safety Manual - rustc - Usage <safety-manual:rustc/usage>`
+     - :doc:`Safety Manual - rustfmt - Usage <safety-manual:rustfmt/usage>`
 
    * - 11.4.4.1
      - See sub items below
+     - See sub items below
 
    * - 11.4.4.1.a
      - :doc:`Safety Manual - Environment <safety-manual:environment>`
+     - See ``rustc`` column
 
    * - 11.4.4.1.b
-     - :doc:`Safety Manual - Tool options <safety-manual:rustc/options>`
+     - :doc:`Safety Manual - rustc - Tool options <safety-manual:rustc/options>`
+     - :doc:`Safety Manual - rustfmt - Tool options <safety-manual:rustfmt/options>`
 
    * - 11.4.4.1.c
-     - :doc:`Evaluation Report - Use cases <evaluation-report:rustc/use-cases>`
+     - :doc:`Evaluation Report - rustc - Use cases <evaluation-report:rustc/use-cases>`
+     - :doc:`Evaluation Report - rustfmt - Use cases <evaluation-report:rustfmt/use-cases>`
 
    * - 11.4.4.1.d
      - :doc:`Safety Manual - Environment <safety-manual:environment>`
+     - See ``rustc`` column
 
    * - 11.4.4.1.e
      - :doc:`Qualification Report - Acceptability statement <qualification-report:statement>`
+     - N/A
```

Comment:
I wouldn't go for an N/A here - the analysis, that all failures are detected in the process is valid for all ASIL levels, so stating somewhere that this is valid up to ASIL D would be more correct than saying N/A.

Reply:
I added that to the ISO tool classification and linked that section.